HISTORY.rst

```diff
@@ -2,8 +2,16 @@
 History
 =======
 
-Latest
-------
+v.0.9.3
+-------
+
+* Released because of a labelling issue. See 0.9.2 for details.
+
+v.0.9.2
+--------
+* Updated documentation
+* Cleaned up examples
+* Added (*)_properties as a property to all components
```

Comment:
What does this mean?

Reply:
eeks, not sure why this PR is still open. Closing. I did two bumpversions, because of some confusion whether anything happened at all after the first bumpversion.